README.md

```diff
@@ -44,13 +44,23 @@ Our chapter's leadership team includes:
 
 Contact [@bobjiang](https://github.com/bobjiang) and  [@daijiale](https://github.com/daijiale)☝️ if you are interested in joining the team! 
 
-## Events Review
+## Events 
 
-### About [LibP2P](https://libp2p.io)
+### Upcoming
+
+We hope the upcoming events could be design and held by community enthusiasts together ，so we specifically open the discursive issue for upcoming event planning ： https://github.com/ProtoSchool/beijing/issues/6 
```

Comment:
Suggestion to address small grammar issues: "We hope that upcoming events will be designed and held by community enthusiasts together, so we've opened a discussion issue for upcoming event planning"

Reply:
Ohhh!  Thanks a lot , I fixed it #https://github.com/ProtoSchool/beijing/pull/7/commits/fac5abfd689a074a5643930e3b1be22242f0d405